src/cache.ts

```diff
@@ -41,28 +47,55 @@ function getCache(): Record<string, string> {
   }
 }
 
+/**
+ * Clean cache
+ */
+export async function cleanCache() {
+  if (await exists(BASE_PATH)) {
+    await Deno.remove(BASE_PATH, { recursive: true });
+  }
+}
+
+async function isQuietInstall() {
+  const { state } = await Deno.permissions.query({
+    name: "env",
+    variable: "ASTRAL_QUIET_INSTALL",
+  });
+  if (state === "granted") {
+    return `${Deno.env.get("ASTRAL_QUIET_INSTALL")}` === "1";
```

Comment:
I've seen alternatives to 1. Maybe just checking that it isn't 0 or false could work?

Reply:
I'll change it to supported YAML booleans for falsy values `/^(0|[Nn]o?|NO|[Oo]ff|OFF|[Ff]alse|FALSE)$/`, should cover most cases 👍